### PR TITLE
Adding test harness for Kustomize targets

### PR DESCRIPTION
**Pull request by ashahba**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/59)

### CODE REVIEW

File: hack/gen-test-targets.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 #
-# gen-test-targets will generate units tests under tests/ for all directories that 
+# gen-test-targets will generate units tests under tests/ for all directories that
 # have a kustomization.yaml. This script first finds all directories and then calls
 # gen-test-target to generate each golang unit test.
 # The script is based on kusttestharness_test.go from kubernetes-sigs/kustomize/pkg/target
```
Comment (hougangliu) [inline hack/gen-test-targets.sh]: kubernetes-sigs/kustomize/pkg/kusttest/kusttestharness.go is the current path

File: hack/gen-test-target.sh [hunk 2]
```diff
@@ -1,25 +1,23 @@
 #!/usr/bin/env bash
 #
-# gen-test-target will generate a golang testcase using the 
+# gen-test-target will generate a golang testcase using the
 # kustomize test-harness that is used from kerbernetes-sigs/kustomize/pkg/target/kusttestharness_test.go
```
Comment (hougangliu) [inline hack/gen-test-target.sh]: kubernetes-sigs/kustomize/pkg/kusttest/kusttestharness.go is the current path

**Comment by ashahba** [comment]
I can't find logs what went wrong. I'm seeing `No such object: kubernetes-jenkins/pr-logs/pull/kubeflow_manifests/59/kubeflow-manifests-presubmit/1126881724291092480` so I guess maybe `prow` is still not fully setup.

**Pull request merged**